DuckDuckGo/AppDelegate+AppDeepLinks.swift

```diff
@@ -22,6 +22,7 @@ import Core
 
 extension AppDelegate {
 
+    // swiftlint:disable:next cyclomatic_complexity
```

Comment:
I don't usually like to disable this one, but this function is specifically designed to be a giant switch statement that handles a large number of options, and I don't think it suffers because of it since each option just calls through to a single function.